app/templates/pages.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="ui segment">
+  <div class="ui grid">
+    <div class="sixteen wide column">
+      <h1 class="ui header">{{model.name}}</h1>
+      <p>{{sanitize model.description}}</p>
```

Comment:
Don't put a p tag here. model.description would have p tags of its own. p inside a p is wrong html semantics. Just put it inside a div tag.